joint_subscriber_topic: "/j2n7s300/joint_states"
torque_publisher_topic: "/j2n7s300/joint_group_effort_controller/command"
finger_publisher_topic: "/j2n7s300/finger_group_position_controller/command"
urdf_file: "/home/alex/Documents/Mcgill/Graduate/COMP-597/workspace/src/comp597-coursework-f2021/inverse_dynamics_controller/urdf/no-gripper.urdf"
use_nullspace: true
num_joints : 7
num_effort_joints : 7
end_effector_joint_id : 7
control_freq : 500
robot:
  init_joint_config : [0, 2.9, 0, 1.3, -2.07, 1.4, 0]
  finger_target_config : [1, 0.3, 1, 0.3, 1, 0.3]
  nullspace_target_config: [0.0, 2.5, 0.0, 1.0, 0.0, 1.0, 0.0]
  torque_limit : [40, 80, 40, 40, 20, 20, 20]
planner:
  attractive_gain: 
      [10, 0, 0,
       0, 10, 0,
       0, 0, 10]
controller:
  task_stiffness: 
      [50, 0, 0,
       0, 50, 0,
       0, 0, 50]
  task_damping: 
      [10, 0, 0,
       0, 10, 0,
       0, 0, 10]
  joint_stiffness: 
      [10, 0, 0, 0, 0, 0, 0,
       0, 10, 0, 0, 0, 0, 0,
       0, 0, 10, 0, 0, 0, 0,
       0, 0, 0, 10, 0, 0, 0,
       0, 0, 0, 0, 5, 0, 0,
       0, 0, 0, 0, 0, 5, 0,
       0, 0, 0, 0, 0, 0, 5]
  joint_damping: 
      [2, 0, 0, 0, 0, 0, 0,
       0, 2, 0, 0, 0, 0, 0,
       0, 0, 2, 0, 0, 0, 0,
       0, 0, 0, 2, 0, 0, 0,
       0, 0, 0, 0, 1, 0, 0,
       0, 0, 0, 0, 0, 1, 0,
       0, 0, 0, 0, 0, 0, 1]
  nullspace_attractive:
      [0.1, 0, 0, 0, 0, 0, 0,
       0, 0.1, 0, 0, 0, 0, 0,
       0, 0, 0.1, 0, 0, 0, 0,
       0, 0, 0, 0.1, 0, 0, 0,
       0, 0, 0, 0, 0.1, 0, 0,
       0, 0, 0, 0, 0, 0.1, 0,
       0, 0, 0, 0, 0, 0, 0.1]